src/gui/ApplicationSettingsWidget.cpp

```diff
@@ -334,6 +336,48 @@ void ApplicationSettingsWidget::saveSettings()
     }
 }
 
+void ApplicationSettingsWidget::resetSettings()
+{
+    // Confirm reset
+    auto ans = MessageBox::question(this,
+                                    tr("Reset Settings?"),
+                                    tr("Are you sure you want to reset all general and security settings to default?"),
+                                    MessageBox::Reset | MessageBox::Cancel,
+                                    MessageBox::Reset);
```

Comment:
Shall we default to the non-destructive option, cancel?

Reply:
Agree, thank you